frontend/src/components/AccountReceiveTableWithdrawConfirmation.vue

```diff
@@ -108,7 +108,12 @@ import { UserAnnouncement } from '@umbra/umbra-js';
 import { FeeEstimate } from 'components/models';
 import { formatAddress, toAddress } from 'src/utils/address';
 import { BigNumber, formatUnits } from 'src/utils/ethers';
-import { getEtherscanUrl, getGasPrice, round, roundTokenAmount } from 'src/utils/utils';
+import {
+  getEtherscanUrl,
+  getGasPrice,
+  humanizeTokenAmount,
+  roundReceivableAmountAfterFees,
+} from 'src/utils/utils';
```

Comment:
If I run `yarn prettier` in the frontend folder, this gets collapsed to a single line (and there's a handful of other changes too, some in unrelated files). It should run in commit hooks, so odd that it didn't, but either let's remember to run prettier before merging this PR. (I'm intentionally not doing it now just to reduce diff noise from other, unrelated formatting changes)